### ISSUE TITLE
h2o init. action breaks with 1.5-ubuntu

### ISSUE THREAD

**Issue by bradmiro** [GoogleCloudDataproc/initialization-actions#774]
Integration test fails with 1.5-ubuntu, haven't fully figured out what's wrong. The testing script itself may just need an update:
```
Traceback (most recent call last):
  File "/tmp/43bb2a99b75d4097a2df49c8d504a63f/sample-script.py", line 6, in <module>
    hc = H2OContext.getOrCreate()
  File "/opt/conda/default/lib/python3.7/site-packages/ai/h2o/sparkling/H2OContext.py", line 89, in getOrCreate
    selected_conf = H2OConf()
  File "/opt/conda/default/lib/python3.7/site-packages/ai/h2o/sparkling/H2OConf.py", line 35, in __init__
    self._jconf = _jvm.org.apache.spark.h2o.H2OConf()
  File "/usr/lib/spark/python/lib/py4j-0.10.7-src.zip/py4j/java_gateway.py", line 1525, in __call__
  File "/usr/lib/spark/python/lib/pyspark.zip/pyspark/sql/utils.py", line 63, in deco
  File "/usr/lib/spark/python/lib/py4j-0.10.7-src.zip/py4j/protocol.py", line 328, in get_return_value
py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.h2o.H2OConf.
: java.lang.NoClassDefFoundError: org/apache/spark/internal/Logging$class
	at ai.h2o.sparkling.utils.SparkSessionUtils$.<init>(SparkSessionUtils.scala:32)
	at ai.h2o.sparkling.utils.SparkSessionUtils$.<clinit>(SparkSessionUtils.scala)
	at org.apache.spark.h2o.H2OConf.<init>(H2OConf.scala:55)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
	at py4j.Gateway.invoke(Gateway.java:238)
	at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
	at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
	at py4j.GatewayConnection.run(GatewayConnection.java:238)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: org.apache.spark.internal.Logging$class
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	... 14 more
```

**Comment by bradmiro** [comment]
Tracking this with the h2o folks [here](https://github.com/h2oai/sparkling-water/issues/2239).

@medb could this potentially be related to something on our side?

**Comment by medb** [comment]
Not sure about this, we use the same Spark version and build process on Datparoc 1.4 and 1.5, so I would expect that this is something wrong with H20

**Comment by bradmiro** [comment]
Hmmm, ok. My initial guess is this is related to Scala versioning, but this also passes on Dataproc 2.0 as they have a Scala 2.12/Spark 3.0 release. I've been trying to custom build w/ Spark 2.4 + Scala 2.12 which they claim is supported but haven't made much progress here.

**Comment by bradmiro** [comment]
@medb I've spent a considerable amount of time looking into this and haven't really made any progress. Do you have any issue with blocking installs on 1.5? And insist to use (version <= 1.4 | 2.0 <= version)

**Comment by medb** [comment]
Yes, let's skip it for 1.5 image until there customers requests to fix h2o on 1.5-ubuntu images. Also, does it work on 1.5 Debian image? If, so, can we skip only 1.5 Ubuntu images?

**Comment by bradmiro** [comment]
Sounds good. This doesn't work on either 1.5 image, so we should skip it altogether.
